outputs.tf

```
@@ -1,31 +1,28 @@
-output "cluster" {
-  value = google_container_cluster.engineering.name
-}
-
-output "host" {
-  value     = google_container_cluster.engineering.endpoint
-  sensitive = true
-}
-
-output "cluster_ca_certificate" {
-  value     = base64decode(google_container_cluster.engineering.master_auth.0.cluster_ca_certificate)
-  sensitive = true
+output "aws_region" {
+  description = "AWS region for all resources"
+  value       = var.aws_region
 }
 
-output "username" {
-  value     = google_container_cluster.engineering.master_auth.0.username
-  sensitive = true
-}
-
-output "password" {
-  value     = google_container_cluster.engineering.master_auth.0.password
-  sensitive = true
+output "cluster" {
```

```suggestion
output "cluster_id" {
```

So it's more descriptive in the downstream workspaces